### PR TITLE
Committees subpages 

### PR DESCRIPTION
**Pull request by jonibim**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: cosmos/settings.py [hunk 1]
```diff
@@ -106,16 +106,11 @@
     "CLOSURE_BINARY": "node_modules/.bin/google-closure-compiler",
     "STYLESHEETS": {
         "cosmos": {
-            "source_filenames": {"cosmos/css/registration.css", "cosmos/css/core.css"},
```
Comment (maximekabel) [inline cosmos/settings.py:109]: Why did you remove this? none of the css gets loaded anymore

**Review by maximekabel — CHANGES_REQUESTED** [review]

File: apps/users/templates/committees/overview.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    <span class="visually-hidden">Previous</span>
+                </button>
+                <button class="carousel-control-next" type="button" data-bs-target="#carouselCommittees"
+                        data-bs-slide="next">
+                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                    <span class="visually-hidden">Next</span>
+                </button>
+            </div>
+        </div>
+        <div class="row">
```
Comment (maximekabel) [inline apps/users/templates/committees/overview.html:49]: For some reason this row has some extra flex padding on the right side which means that it's not inline with the carousel

Comment (dasyad00) [reply apps/users/templates/committees/overview.html]: fixed

**Review by maximekabel — APPROVED** [review]
